draft-fossati-tls-attestation.md

```diff
@@ -183,8 +183,6 @@ encoded. This is accomplished by companion specifications.
 
 The following terms are used in this document:
 
-{: vspace="0"}
-
 TLS Identity Key (TIK): : A cryptographic key used by one of the peers to
```

Comment:
the problem is not with `vspace`.  You need to put a NL after the first `:` in L188

Reply:
Ahh, I see...